### DISCUSSION TITLE
Fade the colors so as to use as a background image

### DISCUSSION THREAD

**Discussion by suntong** [ImageMagick/ImageMagick#2488] [Development] [answered]
How to fade a vivid-colored image so as to use as a background image?

It's very hard to explain, but I've just found a perfect example:

![image](https://user-images.githubusercontent.com/422244/91734554-12f77980-eb79-11ea-8880-68d2e8d87ea3.png)

The image on the south-west side was a vivid-colored image before, but all its colors are faded now, which make it a good candidate as a background image to write texts on, yet, it till gives a sense to people how it looked like before.

How to do that with ImageMagick? To demo, please fade our vivid-colored ImageMagick logo, so as to use it as a background image to write text one. Thx!

**Comment by fmw42** [comment, marked as answer]
You fade the colors by mixing with white. You can do that by using -fill white -colorize 30% for example.

`convert lena.png \( -clone 0 -resize 50% -fill white -colorize 30% \) -gravity southwest -compose over -composite result.png`


![result](https://user-images.githubusercontent.com/13458218/91744085-c22e5380-eb6d-11ea-9142-62048bd1b075.png)
